src/components/dashboard/recentTransactions/recentTransactions.css

```diff
@@ -53,7 +53,15 @@
     }
   }
 
+  & .viewAllLink {
+    margin-top: 22px;
+  }
+
   & .viewAllBtn {
-    margin-top: 15px;
+    @mixin contentNormal bold;
+
+    width: 85px;
+    height: 36px;
```

Comment:
@osvaldovega we have classes created my @massao  for button sizes
https://github.com/LiskHQ/lisk-hub/blob/004fdc7e36e648868762bad596ac13e950741279/src/components/toolbox/buttons/demo.js#L32-L35

Reply:
Also, I'm not sure if the contentNormal mixin is needed in this case, since all buttons, should already have the correct font style, so it's easier to update in the future if needed

Reply:
Different button sizes have different font size, so I understand that Osvaldo needed the mixin when not using the size className. With the className, it's not needed anymore.